src/main/java/com/easypost/service/CarrierAccountService.java

```diff
@@ -33,14 +33,17 @@ public class CarrierAccountService {
      */
     public CarrierAccount create(final Map<String, Object> params) throws EasyPostException {
         String type = (String) params.get("type");
+        Map<String, Object> wrappedParams = new HashMap<String, Object>();
+
         if (type == null) {
             throw new MissingParameterError(
                     String.format(Constants.ErrorMessages.MISSING_REQUIRED_PARAMETER, "carrier account type"));
+        } else if (type.equals("UpsAccount")) {
```

Comment:
Need to account for "UpsMailInnovationsAccount" and 
"UpsSurepostAccount" as well